frontend/tailwind.config.js

```diff
@@ -15,35 +15,14 @@ module.exports = {
       white: '#fff',
       gray: '#D8D8D8',
       primary: '#D14746',
+      'font-primary': '#050117',
+      'font-secondary': '#060118',
+      'link-primary': '#004CD7'
```

Comment:
we shouldn't name the colors with the context. This color can be used for other elements

Reply:
then it would be here multiple times in that case. eg maybe you would have 
`button-primary: '#D14746',
'primary: '#D14746'`
If the brand's primary colour and the primary button colour are the same. This way the names are semantic.

Reply:
Normally I'd use something that can be used more globally and don't depend on the context, e.g.:
`
primary (commonly extracted from the brand colors, used for the most outstanding elements).
secondary (also regularly extracted from the brand colors, for other outstanding elements).
tertiary (a color used in many elements through the website, different from the accent. Sometimes also extracted from the brand colors).
accent (commonly used in links, buttons, hovers).
gray-0 to gray-100 (based on the black color percentage, gray-0 would be white and gray-100 would be black. In practice 0 and 100 values shouldn't be used).
other color names based on the color itself.
`
Something I used on some projects like WF was to create some additional variables after the ones above, that defined the context, e.g. `$link: $accent;`, `$button-primary: $accent`, `$button-secondary: $secondary`, etc.

Reply:
Also the properties in tailwind.config.js shouldn't be wrapped by quotation marks, unless they are numbers.

Reply:
My philosophy was that by doing it this way you can have some predefined fields in this config which you fill and everything just works. Eg link-primary will be applied to <a>'s by default, so you would just update this in the config and all your links will be styled, no more other code changes required. Maybe I'm being too ambitious, what do you think?

I'll check on the quotes, I thought it was necessary when you have a - in the name.

Reply:
> Something I used on some projects like WF was to create some additional variables after the ones above, that defined the context, e.g. $link: $accent;, $button-primary: $accent, $button-secondary: $secondary, etc.
-- I like this approach in SASS, but I'm not sure how it would be possible in TW. Can you think of a way?

Reply:
Sorry, you're right, quotes are necessary when you have a - in the name.

I like your approach, we could combine both approaches, having some abstract names like primary and secondary, and also some contextual names, like link-primary, with the abstract names applied. What do you think if we use [this method](https://share.getcloudapp.com/YEudXyAP)?

Reply:
Rather than use css variables i think we can do this in JS. Something like:
![image](https://user-images.githubusercontent.com/5068959/71683807-31db0d00-2d59-11ea-92a2-ef77bcde957f.png)
and then:
![image](https://user-images.githubusercontent.com/5068959/71683859-533bf900-2d59-11ea-9790-0198562c3fd3.png)